src/decoder_wildmidi.cpp

```diff
@@ -253,9 +253,10 @@ WildMidiDecoder::WildMidiDecoder(const std::string file_name) {
 	}
 	Output::Debug("WildMidi: Using %s as configuration file...", config_file.c_str());
 
+	Output::Debug("WildMidi: Calling WildMidi_Init(%s, %d, %d)...", config_file.c_str(), WILDMIDI_FREQ, WILDMIDI_OPTS);
```

Comment:
Even the debug messages should be non-developer readable imo. (to understand the log line you have look up which arguments WildMidi_Init takes)
Also the config file information is redundant (see 2 lines above)

Reply:
Yep, this line seems redundant, as in error case the next line will report it.

Reply:
Removed the line. It is redundant.